packages/react/context-menu/src/ContextMenu.tsx

```diff
@@ -103,7 +103,7 @@ const ContextMenuTrigger = React.forwardRef<ContextMenuTriggerElement, ContextMe
     React.useEffect(() => clearLongPress, [clearLongPress]);
 
     return (
-      <ContentContext.Provider value={false}>
+      <ContentProvider isInsideContent={false}>
```

Comment:
This is arguable more readable now too 👍

Reply:
I thought that as well ... 🎉